projects/python/perception/facial_expression_recognition/image_based_facial_emotion_estimation/benchmark_esr.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2020-2022 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
```

Comment:
A benchmark script is provided but no information seems to be present in the documentation, are there any results available yet?